Source/Api/Controllers/EventController.cs

```diff
@@ -123,39 +123,38 @@ public class EventController : RepositoryApiController<IEventRepository, Persist
         [Route]
         [ResponseType(typeof(List<PersistentEvent>))]
         public async Task<IHttpActionResult> GetAsync(string filter = null, string sort = null, string time = null, string offset = null, string mode = null, int page = 1, int limit = 10) {
-            var organizations = (await GetAssociatedOrganizationsAsync(_organizationRepository)).Where(o => !o.IsSuspended).ToList();
+            var organizations = await GetAssociatedActiveOrganizationsAsync(_organizationRepository);
             if (organizations.Count == 0)
-                return Ok(Enumerable.Empty<PersistentEvent>());
+                return Ok(EmptyModels);
 
             var ti = GetTimeInfo(time, offset, organizations.GetRetentionUtcCutoff());
             var sf = new ExceptionlessSystemFilterQuery(organizations);
             return await GetInternalAsync(sf, ti, filter, sort, mode, page, limit);
         }
 
-        private async Task<IHttpActionResult> GetInternalAsync(IExceptionlessSystemFilterQuery sf, TimeInfo ti, string userFilter = null, string sort = null, string mode = null, int page = 1, int limit = 10, bool usesPremiumFeatures = false) {
+        private async Task<IHttpActionResult> GetInternalAsync(IExceptionlessSystemFilterQuery sf, TimeInfo ti, string filter = null, string sort = null, string mode = null, int page = 1, int limit = 10, bool usesPremiumFeatures = false) {
             page = GetPage(page);
             limit = GetLimit(limit);
             var skip = GetSkip(page, limit);
             if (skip > MAXIMUM_SKIP)
-                return Ok(Enumerable.Empty<PersistentEvent>());
+                return Ok(EmptyModels);
 
             var sortBy = GetSort(sort);
             var options = new PagingOptions { Page = page, Limit = limit };
 
-            var pr = QueryProcessor.Process(userFilter);
+            var pr = QueryProcessor.Process(filter);
```

Comment:
We need to work on getting rid of this, but doesn't have to be done right away. It should be implemented as settings on the query parser for the elastic type / repo.